File: snippets/python_snippets/lnurl_auth.py
import breez_sdk
import logging

sdk_services = breez_sdk.BlockingBreezServices

def auth():
    # ANCHOR: lnurl-auth
    # Endpoint can also be of the form:
    # keyauth://domain.com/auth?key=val
    lnurl_auth_url = "lnurl1dp68gurn8ghj7mr0vdskc6r0wd6z7mrww4excttvdankjm3lw3skw0tvdankjm3xdvcn6vtp8q6n2dfsx5mrjwtrxdjnqvtzv56rzcnyv3jrxv3sxqmkyenrvv6kve3exv6nqdtyv43nqcmzvdsnvdrzx33rsenxx5unqc3cxgeqgntfgu"

    try:
      parsed_input = breez_sdk.parse_input(lnurl_auth_url)     
      if isinstance(parsed_input, breez_sdk.InputType.LN_URL_AUTH):      
        req = breez_sdk
        result = sdk_services.lnurl_auth(parsed_input.data)        
      if result.is_ok():
        logging.info("Successfully authenticated")
      else:
        logging.error("Failed to authenticate")
    except Exception as error:
        logging.error(error)
        raise
    # ANCHOR_END: lnurl-auth

File: snippets/dart_snippets/lib/lnurl_auth.dart
import 'package:breez_sdk/breez_sdk.dart';
import 'package:breez_sdk/bridge_generated.dart';

Future<void> lnurlAuth() async {
  // ANCHOR: lnurl-auth
  /// Endpoint can also be of the form:
  /// keyauth://domain.com/auth?key=val
  String lnurlAuthUrl =
      "lnurl1dp68gurn8ghj7mr0vdskc6r0wd6z7mrww4excttvdankjm3lw3skw0tvdankjm3xdvcn6vtp8q6n2dfsx5mrjwtrxdjnqvtzv56rzcnyv3jrxv3sxqmkyenrvv6kve3exv6nqdtyv43nqcmzvdsnvdrzx33rsenxx5unqc3cxgeqgntfgu";

  InputType inputType = await BreezSDK().parseInput(input: lnurlAuthUrl);
  if (inputType is InputType_LnUrlAuth) {
    LnUrlCallbackStatus result = await BreezSDK().lnurlAuth(reqData: inputType.data);
    if (result is LnUrlCallbackStatus_Ok) {
      print("Successfully authenticated");
    } else {
      print("Failed to authenticate");
    }
  }
  // ANCHOR_END: lnurl-auth
}
